src/java.base/share/classes/java/net/ServerSocket.java

```diff
@@ -855,22 +855,22 @@ public int getSoTimeout() throws IOException {
      * {@code SocketAddress} if there is a connection in the
      * timeout state involving the socket address or port.
      * <p>
-     * Enabling {@link SocketOptions#SO_REUSEADDR SO_REUSEADDR} prior to
+     * Enabling {@link StandardSocketOptions#SO_REUSEADDR SO_REUSEADDR} prior to
      * binding the socket using {@link #bind(SocketAddress)} allows the socket
      * to be bound even though a previous connection is in a timeout state.
      * <p>
      * When a {@code ServerSocket} is created the initial setting
-     * of {@link SocketOptions#SO_REUSEADDR SO_REUSEADDR} is not defined.
+     * of {@link StandardSocketOptions#SO_REUSEADDR SO_REUSEADDR} is not defined.
      * Applications can use {@link #getReuseAddress()} to determine the initial
-     * setting of {@link SocketOptions#SO_REUSEADDR SO_REUSEADDR}.
+     * setting of {@link StandardSocketOptions#SO_REUSEADDR SO_REUSEADDR}.
      * <p>
-     * The behaviour when {@link SocketOptions#SO_REUSEADDR SO_REUSEADDR} is
+     * The behaviour when {@link StandardSocketOptions#SO_REUSEADDR SO_REUSEADDR} is
```

Comment:
I suppose the main question here is whether the description really needs to link to SO_REUSEADDR five times, it seems a bit excessive. In cases like this I tend to just have the first usage link, others do it differently.

Reply:
Hello Alan, I too typically follow the process of linking once and then using `{@code SO_REUSEADDR}`. I let it stay in this form since it was already that way. I'll go ahead and update it to link only once.

Reply:
Done - I've updated the PR to remove the repeated links.